src/Symfony/Component/Security/Acl/Dbal/Schema.php

```diff
@@ -61,94 +77,11 @@ public function addToSchema(BaseSchema $schema)
     }
 
     /**
-     * Adds the class table to the schema.
-     */
-    protected function addClassTable()
-    {
-        $table = $this->createTable($this->options['class_table_name']);
-        $table->addColumn('id', 'integer', array('unsigned' => true, 'autoincrement' => 'auto'));
-        $table->addColumn('class_type', 'string', array('length' => 200));
-        $table->setPrimaryKey(array('id'));
-        $table->addUniqueIndex(array('class_type'));
-    }
-
-    /**
-     * Adds the entry table to the schema.
-     */
-    protected function addEntryTable()
```

Comment:
What do you suggest? I must maintain a way of adapting the methods that create the tables so they can have different codes for specific databases. Should I change the PR for BC Breaks?